packages/smooth_app/lib/pages/onboarding/country_selector.dart

```diff
@@ -3,11 +3,18 @@ import 'package:iso_countries/iso_countries.dart';
 import 'package:openfoodfacts/utils/CountryHelper.dart';
 import 'package:smooth_app/data_models/user_preferences.dart';
 import 'package:smooth_app/database/product_query.dart';
-import 'package:smooth_ui_library/util/ui_helpers.dart';
 
 /// A selector for selecting user's country.
 class CountrySelector extends StatefulWidget {
-  const CountrySelector();
+  const CountrySelector({
+    this.initialCountryCode,
+    this.inputDecoration,
+    this.padding,
```

Comment:
Optional comment: It might be better to just add a padding at the call site rather than passing it as an argument.

Reply:
I saw that, but the padding would then apply to `Text('fatal error')` and `CircularProgressIndicator` too.

Reply:
hmm I see